src/Script/Script.ts

```diff
@@ -32,9 +31,19 @@ export class Script {
    */
   dependencies: Map<ScriptURL, Script> = new Map();
 
-  constructor(fn = "", code = "", server = "") {
+  get content() {
+    return this.code;
+  }
+  set content(code: string) {
+    const newCode = Script.formatCode(code);
```

Comment:
This change (minorly) broke how my scripts work. Let me explain.

I have an installer script that updates all my code. Yes, I should probably use Viteburner, but that's not the point here. Before it writes, it checks if the content is the same, and skips writing if it differs. I can use that as a quick sanity check to see that it's updating the expected set of files.

My code has a newline at EOF, because this is The Proper Convention. However, BB's editor strips that newline. No big deal, since I don't use the in-game editor, but it would show files as modified always, even when first opening them.

Now, however, *writing* the files changes the contents, and so my installer script flags them as out-of-date every time. It's impossible to actually write the code as it was written. IMO, this is a bug - only the editor should be reformatting the code.

Reply:
Yeah I see what you mean. Honestly there's not a great reason for the code editor to do it either. Sure we can trim off the trailing whitespace to save a few bytes on the save file, but it's not really meaningful and it's just a weird unexpected behavior in general.